tests/integration/template_complete_run.cfg

```diff
@@ -25,9 +25,9 @@ years = "1850:1854:2", "1850:1854:4",
   vars = "PRECT"
 
 [ts]
-active = True
+active = 4
```

Comment:
Causes `ValueError: Invalid value 4 for 'active'` at runtime rather than a failed configuration validation.